log4j-core/src/main/java/org/apache/logging/log4j/core/util/NetUtils.java

```diff
@@ -79,6 +84,45 @@ public static String getLocalHostname() {
         }
     }
 
+    public static List<String> getLocalIps() {
+        List<String> localIps = new ArrayList<>();
+        localIps.add("localhost");
+        localIps.add("127.0.0.1");
```

Comment:
Perhaps IPv6 as well `[::1]`? (brackets may or may not be necessary depending how we extract the host)

Reply:
I could have but this pre-seeding really isn't necessary as they show up in the while loop anyway.